Prace_domowe/Praca_domowa2/Grupa2/Jakub_Koziel/Praca domowa 2.ipynb

```diff
@@ -0,0 +1,2153 @@
+{
```

Comment:
<p>Dobry pomysł żeby to przedstawić na wykresie.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/117/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>